ca_tools/action/FollowLine.action

```diff
@@ -1,10 +1,10 @@
 # Goal
-int32 amount
+int32 goal_seconds
```

Comment:
Use a duration msg.

Reply:
For the goal I think the `int32` is right, since it would be an easier argument to pass, for the other messages tho I think duration is the answer